src/vs/workbench/contrib/notebook/common/notebookCommon.ts

```diff
@@ -37,13 +37,15 @@ export const NOTEBOOK_DISPLAY_ORDER = [
 	'application/javascript',
 	'text/html',
 	'image/svg+xml',
+	Mimes.latex,
```

Comment:
This ordering actually looks off from the Jupyter ordering. I see this in classic notebooks and jupyterlab:
https://github.com/jupyter/notebook/blob/41f148395c2b056998768423257d9c8edb4244ec/notebook/static/notebook/js/outputarea.js#L272
https://github.com/jupyterlab/jupyterlab/blob/dbdefed9db9332381fe4104bdf53ec314451951f/packages/rendermime/src/factories.ts#L31

Reply:
I'm seeing markdown->latex->svg in that priority order if I'm reading those right. So SVG also looks misplaced.

Reply:
This isn't for ipynb files, there's a seprate ordering for ipynb notebooks.